solr/core/src/java/org/apache/solr/metrics/prometheus/PrometheusCoreExporterInfo.java

```diff
@@ -30,4 +32,7 @@ enum CoreCategory {
     INDEX,
     CORE
   }
+
+  String PROMETHEUS_CLOUD_CORE_REGEX = "^core_(.*)_(shard[0-9]+)_(replica_n[0-9]+)$";
+  Pattern PROMETHEUS_CLOUD_CORE_PATTERN = Pattern.compile(PROMETHEUS_CLOUD_CORE_REGEX);
```

Comment:
But isn't the word "PROMETHEUS" in here actually inaccurate?  This is parsing existing internal names, it's not an external prometheus exported pattern.  I think you could just drop this part.

Reply:
You're right dropped `PROMETHEUS` in the prefix and also removed the `PROMETHEUS_CLOUD_CORE_REGEX` constant